src/main/java/net/jonathangiles/tools/teenyhttpd/json/JsonAlias.java

```diff
@@ -0,0 +1,12 @@
+package net.jonathangiles.tools.teenyhttpd.json;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface JsonAlias {
```

Comment:
We need to do a good JavaDoc pass at some point before it becomes technical debt!